Modelica/Mechanics/Rotational/Components/LossyGear.mo

```diff
@@ -26,7 +26,7 @@ model LossyGear
   SI.AngularAcceleration a_a
     "Angular acceleration of flange_a with respect to support";
 
-  Real interpolation_result[1, 4]
+  Real interpolation_result[1, nCol-1]
```

Comment:
It is not clear to me that using a constant is better here. You now see `nCol-1` and also a description that describes **four** columns; so I'm more left wondering whether they are the same or not. As far as I understand there is no plan to make nCol variable. If we are going with such a constant it would to me be more logical to have nValueCols=4, and replace nCol and nCol-1 with nValueCols+1 and nValueCols; as it seems more normal to add columns than to remove them.

Reply:
The introduction of nCol is rather intended for consistency in dimension of lossTable and constructor of tableID. But it shall also be consistent for interpolation_result. So I simply took a short named constant for it.

I have no preference for it - we can also call it e.g. nValue=4.

Reply:
I propose to remove constant nCol again. It is protected, but SimulationX displays it as dimensions of lossTable where nCol cannot be deduced at all.

Reply:
Ok, I will revert the changes on it.

Reply:
Reverted with 8a2bb0a7a3d336c5b49537e3de01feef984d3b6d